tools/ci/tcdownload.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import os
+
+import requests
+
+import github
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--ref", action="store", default="master",
+                        help="branch or commit to fetch logs for")
+    parser.add_argument("--artifact-name", action="store", default="wpt_report.json.gz",
+                        help="Log type to fetch")
+    parser.add_argument("--repo-name", action="store", default="web-platform-tests/wpt",
+                        help="GitHub repo name in the format owner/repo")
+    parser.add_argument("--token-file", action="store",
```

Comment:
Can we either make this work without a token or make this required? Currently if you fail to to pass `--token-file` you get a TypeError, which isn't a great UX.

Reply:
The thinking was that in the future it could read the token from an environment variable or somewhere else so making it a required argument will result in API churn. But sure, the error should be nicer.

Reply:
I mean, in the common case, we shouldn't hit the rate limit here even without a token, no?

Reply:
I though it required authentication, but apparently I jsut typoed the URL. Fixed.